int32 point_cnt
uint8[1200] raw_point
